Vvnu/codeeditor
# fix: color scheme and font of "Language" and "Output"

This PR fixes the color scheme of "Language" and "Run code" buttons, and the font of "Language".

Fixes #3  

https://github.com/user-attachments/assets/de69089a-1cd9-4bf8-afd4-dc10cf23b6b2

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- [] Go to CodeEditor page
- [] See the color scheme of "Language" and "Run code" buttons, and the font of "Language".

## Mandatory Tasks
